doc/src/devdocs/external_profilers.md

```diff
@@ -0,0 +1,60 @@
+# External Profiler Support
+
+Julia provides explicit support for some external tracing profilers, enabling you to obtain a high-level overview of the runtime's execution behavior.
+
+The currently supported profilers are:
+- [Tracy](https://github.com/wolfpld/tracy)
+- [ITTAPI](https://github.com/intel/ittapi)
+
+### Adding New Zones
+
+To add new zones, use the `JL_TIMING` macro. You can find numerous examples throughout the codebase by searching for `JL_TIMING`.
```

Comment:
Would be nice to briefly mention the metadata `jl_timing_show` and `jl_timing_printf` functions here